open_humans/templates/email/welcome.txt

```diff
@@ -0,0 +1,34 @@
+Thank you for joining the Open Humans community!
+
+To make sure you get future emails from us, add our email to your “contacts”.
+
+{% if newsletter %}You’ve signed up for news and updates, but without this
+step our emails might be “filtered” by your email client.{% else %}
```

Comment:
I think I'm seeing non-ASCII characters (left and right quotation marks). Would this be an issue, or is non-ASCII fine? (Note, non-ASCII quote marks are also in the HTML email.)